addon/templates/components/frost-modal-binding.hbs

```diff
@@ -4,8 +4,9 @@
     send=(hash
       animation=animation
       body=(component modal
-        classModifier=classModifier
+        class=class
```

Comment:
Does there need to be a corresponding change in code somewhere for this property name change?